src/modules/email/email.service.ts

```diff
@@ -55,4 +59,13 @@ export class EmailService {
       },
     });
   }
+
+  async sendContactInquiryMail(context: { name: string; email: string; message: string; created_at: string }) {
```

Comment:
```suggestion
async sendContactInquiryMail(context: emailContextDto)
```

This is a reusable dto that can also be typed at `src/modules/contactUs/contact-us.service.ts`

Reply:
I see you have a `ContactUsDto` already. That can be used too

Reply:
done @CFCIfe